### PR TITLE
CSS cleanup

### PR DESCRIPTION
**Pull request by XhmikosR**
**By submitting this pull request, I confirm the following:** `{please fill any appropriate checkboxes, e.g: [X]}`

`{Please ensure that your pull request is for the 'devel' branch!}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

Requires #1097 first.

### CODE REVIEW

**Comment by XhmikosR** [comment]
I'll probably drop the last patch because I'm not sure if we'll keep using animate.css forever and it's only a few lines anyway. I'll rebase after #1097 is merged.

**Comment by PromoFaux** [comment]
Rebase please

**Comment by XhmikosR** [comment]
There weren't any conflicts but rebased it anyway.

File: index.php [hunk 1]
```diff
@@ -154,10 +154,10 @@ function getinterval()
         </div>
         <div class="box-body">
           <div class="pull-left" style="width:50%">
-            <canvas id="forwardDestinationPieChart" width="120" height="120" class="extratooltipcanvas"></canvas>
+            <canvas id="forwardDestinationPieChart" width="120" height="120" class="extratooltipcanvas no-user-select"></canvas>
           </div>
           <div class="pull-left" style="width:50%">
-            <div id="forward-destinations-legend" class="chart-legend" class="extratooltipcanvas"></div>
+            <div id="forward-destinations-legend" class="chart-legend" class="extratooltipcanvas no-user-select"></div>
```
Comment (XhmikosR) [inline index.php]: Oops just noticed the duplicate `class`. I'll remove it from here, but I'm not sure if there are more cases.

**Review by PromoFaux — APPROVED** [review]

**Pull request merged**